vlib/v/fmt/tests/multiline_comment_keep.vv

```diff
@@ -1,8 +1,8 @@
 /*
-this is a very long comment
```

Comment:
Please keep the formatting for this file.

Reply:
Hmmm, I talked to Alex about this. I don't know if I misunderstood something?

Reply:
I'll consult with gofmt and do the same.